### DISCUSSION TITLE
I can't login

### DISCUSSION THREAD

**Discussion by Daniellop1** [umami-software/umami#996] [Q&A]
Hey, I just deployed my instance in Vercel but I'm getting this errors while I was trying to log in.
<img width="745" alt="Captura de Pantalla 2022-01-22 a la(s) 10 21 18 p m" src="https://user-images.githubusercontent.com/44546966/150655830-9d466d75-c727-4c58-89de-617356fb12b8.png">

**Comment by mikecao** [comment]
You need to check the Vercel error logs to see what the error is.
